sensu/server.sls

```diff
@@ -187,6 +188,14 @@ uchiwa:
 {{ logship('sensu-api.log',  '/var/log/sensu/sensu-api.log', 'sensu', ['sensu', 'sensu-api', 'log'],  'rawjson') }}
 {{ logship('uchiwa.log',  '/var/log/upstart/uchiwa.log', 'sensu', ['sensu', 'sensu-dashboard', 'uchiwa', 'log'],  'rawjson') }}
 
+/etc/collectd/collectd.conf.d/processes-redis.conf:
+  file.managed:
+    - source: salt://sensu/files/graph-redis.conf
+    - user: root
+    - group: root
+    - watch_in:
+      - service: collectd
```

Comment:
I think for this to work you need to put `require: collectd-confd-clean` (or maybe `require_in`) for this to work and not remove then re-add the file.

Check the salt highstate run - it should run with 0 changes and I suspect right now it won't